### PR TITLE
Add Columns selection element

### PR DESCRIPTION
**Pull request by danielajass**
- add an option to select columns in the table.

### CODE REVIEW

**Comment by mrshu** [comment]
@danielajass would you mind re-running `black` through these changes?

**Comment by danielajass** [comment]
> @danielajass would you mind re-running `black` through these changes?

I did. Now it passed.

**Comment by mrshu** [comment]
Thanks @danielajass!

The only question/suggestion I would have would be whether it would make sense to save the columns to show with the dataset, so that one wouldn't need to pick them at all times. Do you think it would make sense to add something like that?

**Comment by danielajass** [comment]
> The only question/suggestion I would have would be whether it would make sense to save the columns to show with the dataset, so that one wouldn't need to pick them at all times. Do you think it would make sense to add something like that?

@mrshu, good idea! I looked at the code and looks doable. I'll add it.

File: prompterator/utils.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def save_labelled_data():
             c.PROMPT_COMMENT_KEY: st.session_state[c.PROMPT_COMMENT_KEY],
             c.MODEL_KEY: model.name,
             c.PROMPT_CREATOR_KEY: get_prompt_creator(),
+            c.COL_TO_SHOW_KEY: st.session_state["columns_to_show"],
```
Comment (samsucik) [inline prompterator/utils.py]:
```suggestion
            c.COLS_TO_SHOW_KEY: st.session_state[c.COLS_TO_SHOW_KEY],
```

**Review by samsucik — APPROVED** [review]
All looks good; I haven't manually tested Prompterator with the latest version of these changes but I trust you on this as you're essentially making these changes to enable your own work 😄

**Pull request merged**